### PR TITLE
ahmc fixes

### PR DESCRIPTION
**Pull request by sthayashi**

### CODE REVIEW

File: src/samplers/mcmc/ahmc/ahmc_sampleid.jl [hunk 1]
```diff
@@ -28,8 +28,8 @@ AHMCSampleIDVector(contents::NTuple{8,Any}) = StructArray{AHMCSampleID}(contents
 
 AHMCSampleIDVector(::UndefInitializer, len::Integer) = AHMCSampleIDVector((
     Vector{Int32}(undef, len), Vector{Int32}(undef, len),
-    Vector{Int64}(undef, len), Vector{Int64}(undef, len),
-    Vector{Float64}(undef, len), Vector{Int64}(undef, len),
+    Vector{Int64}(undef, len), Vector{Int32}(undef, len),
+    Vector{Float64}(undef, len), Vector{Int32}(undef, len),
     Vector{Bool}(undef, len), Vector{Float64}(undef, len),
 ))
```
Comment (sthayashi) [inline src/samplers/mcmc/ahmc/ahmc_sampleid.jl:34]: Fixes element types to match constructor above. Needed for `AHMCSampleIDVector` to be recognized as a `DensitySampleVector`

**Comment by oschulz** [comment]
Great! Let me know when this can be merged?

**Comment by sthayashi** [comment]
ready to be merged now

**Comment by oschulz** [comment]
Thanks!

**Pull request merged**